database/user_space/db.go

```diff
@@ -21,10 +21,11 @@ const (
 	getUserSpacesQuery                   = `SELECT * FROM user_space;`
 	getUserSpacesBySpaceIDQuery          = `SELECT * FROM user_space WHERE space_id = $1;`
 	getUserSpacesByUserIDQuery           = `SELECT * FROM user_space WHERE user_id = $1;`
-	getUserSpacesBySpaceIDAndUserIDQuery = `SELECT * FROM user_space WHERE user_id = $1 AND space_id = $2;`
+	getUserSpacesByUserIDAndSpaceIDQuery = `SELECT * FROM user_space WHERE user_id = $1 AND space_id = $2;`
 	getUserSpaceValueByIDQuery           = `SELECT value FROM user_space WHERE user_id = $1 AND space_id = $2;`
 
 	getUserSpaceIndirectAdmins = `SELECT GetIndirectSpaceAdmins($1);`
+	checkIsIndirectAdminQuery  = `SELECT EXISTS(SELECT 1 FROM (SELECT GetIndirectSpaceAdmins($2) AS user_id) AS t WHERE user_id = $1);`
```

Comment:
Minor suggestion
It would be nice to make it more readable 
```sql
WITH space_admins AS (
    SELECT GetIndirectSpaceAdmins('bdfd97c5-e9c8-4246-9d4e-b0891a39f018') AS user_id
)
SELECT EXISTS(
               SELECT user_id
               FROM space_admins
               WHERE user_id = '00000000-0000-0000-0000-000000000013') AS user_is_admin;
```
Currently PG with version >= 12 use CTE as "syntax sugar" and does not affect performance 
Stackoverflow [answer](https://stackoverflow.com/a/16467634/356223) related to old days of [PG 9.3](https://www.postgresql.org/support/versioning/)

Other Ref links: [1](https://www.reddit.com/r/PostgreSQL/comments/ehrsft/am_confused_are_using_ctes_to_improve_performance/), [2](https://habr.com/ru/company/postgrespro/blog/451344/)